bigchaindb/pipelines/block.py

```diff
@@ -134,6 +133,27 @@ def delete_tx(self, block):
         return block
 
 
+def tx_collector():
```

Comment:
Didn't do it to be smart. I wanted to keep the deduplication logic separate from the Block pipeline. Coroutine just occurred to me before object, so i dids it... Also, thinking is good for your brain ;)